ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -76,7 +76,7 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-paused
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
-- as: gcp-ipi-workload-identity-rt-fips-f14
+- as: gcp-ipi-oidc-rt-fips-f14
```

Comment:
Shall we update the cron entry as well?

Reply:
Ideally, yes, but because I did not update its frequency, it make no much difference on cron,  maybe later we can update all the crons in a separate PR.